packages/kbn-config-schema/tsconfig.json

```diff
@@ -1,17 +1,21 @@
-{
-  "extends": "../../tsconfig.json",
-  "compilerOptions": {
-    "declaration": true,
-    "declarationDir": "./target/types",
-    "outDir": "./target/out",
-    "stripInternal": true,
-    "declarationMap": true
-  },
-  "include": [
-    "./types/joi.d.ts",
-    "./src/**/*.ts"
-  ],
-  "exclude": [
-    "target"
-  ]
-}
+{

```

Comment:
This and the other config files were updated to use different line endings, please update your git config to preserve the line endings that were in the file: https://help.github.com/articles/dealing-with-line-endings/#global-settings-for-line-endings

Reply:
I also recommend installing the `editorconfig` plugin for your editor so it will automatically pickup configuration files like https://github.com/elastic/kibana/blob/master/.editorconfig